coding/trie.hpp

```diff
@@ -29,12 +31,12 @@ class Iterator
   };
 
   buffer_vector<Edge, 8> m_edge;
-  buffer_vector<TValue, 2> m_value;
+  TValueList m_valueList;
```

Comment:
Would you be so kind to fix typedef in inner Edge structure from EdgeStrT to TEdgeStr (or even to TEdgeLabel)?